Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:jammy AS builder
+LABEL maintainer="TA <ta@lightscale.io>"
+
+RUN echo 'Etc/UTC' > /etc/timezone && \
+    ln -s /usr/share/zoneinfo/Etc/UTC /etc/localtime && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends tzdata && \
+    rm -rf /var/lib/apt/lists/*
+RUN apt update && \
+    apt install -y --no-install-recommends \
+    ca-certificates \
+    curl  \
+    gnupg && \
+    rm -rf /var/lib/apt/lists/*
+RUN curl https://bazel.build/bazel-release.pub.gpg | \
+    apt-key add - && \
+    echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | \
+    tee /etc/apt/sources.list.d/bazel.list
+RUN apt update && \
+    apt install -y --no-install-recommends \
+    bazel \
+    bazel-6.2.1 && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV HOME /root
+RUN apt update && \
+    apt install -y --no-install-recommends \
+    git \
+    make \
+    libssl-dev \
+    zlib1g-dev \
+    libbz2-dev \
+    libreadline-dev \
+    libsqlite3-dev \
+    libncursesw5-dev \
+    xz-utils \
+    tk-dev \
+    libxml2-dev \
+    libxmlsec1-dev \
+    libffi-dev \
+    liblzma-dev \
+    python-is-python3 && \
+    rm -rf /var/lib/apt/lists/*
+RUN curl https://pyenv.run | bash
+RUN CONFIGURE_OPTS=--enable-shared $HOME/.pyenv/bin/pyenv install 3.10.12
+RUN mkdir -p $HOME/bin
+ENV PATH="${HOME}/.pyenv/shims:${HOME}/.pyenv/bin:${HOME}/bin:${PATH}"
+
+COPY . /usr/src/tachyon
+WORKDIR /usr/src/tachyon
+
+RUN bazel build -c opt --config halo2 --//:has_openmp  --//:c_shared_object //scripts/packages/debian/runtime:debian
+RUN bazel build -c opt --config halo2 --//:has_openmp  --//:c_shared_object //scripts/packages/debian/dev:debian
+
+FROM ubuntu:jammy AS runner
+LABEL maintainer="TA <ta@lightscale.io>"
+
+WORKDIR /usr/src/tachyon
```

Comment:
Do we need thsi?

Reply:
Not really. I was just imagining maybe for users running docker container with this image, it would be more convenient if the directory is already changed to /usr/src/tachyon since all the debian packages are there.